ui/tests/e2e/env.js

```diff
@@ -1,11 +1,45 @@
+const core = require('@actions/core');
+
 const MESHERY_SERVER_URL = process.env.MESHERY_SERVER_URL || 'http://localhost:9081';
 const REMOTE_PROVIDER_URL = process.env.REMOTE_PROVIDER_URL || 'http://localhost:9876';
+
+const USER_EMAIL = process.env.REMOTE_PROVIDER_USER_EMAIL;
+const USER_PASSWORD = process.env.REMOTE_PROVIDER_USER_PASSWORD;
+
 const REMOTE_PROVIDER_USER = {
-  email: process.env.REMOTE_PROVIDER_USER_EMAIL || 'test-admin@layer5.io',
-  password: process.env.REMOTE_PROVIDER_USER_PASSWORD || 'test-admin',
+  email: USER_EMAIL || 'test-admin@layer5.io',
+  password: USER_PASSWORD || 'test-admin',
```

Comment:
@jerensl, in addition to testing with a locally accessible remote provider, two new secrets have been created today for use with the publicly hosted remote provider (at https://meshery.layer5.io). I mention this in case those two secrets are helpful. I've tagged you in Slack, so that you might consider their use, if helpful. // @captain-Akshay